src/components/index/DeFiChainEcoSystemSection.tsx

```diff
@@ -39,11 +39,6 @@ export function DeFiChainEcoSystemSection(): JSX.Element {
   const { t } = useTranslation("page-index");
   const sectionTitle = t("EcosystemSection.title").split(" ");
 
-  // TODO
```

Comment:
commenting for line 37, curious why dont we declare `const [hoverState, setHoverState] = useState<string | undefined>(undefined);` into `<Card>` component instead?

Reply:
Missed this out from previous component refactor, thanks~